web/src/features/fba/pages/FireBehaviourAdvisoryPage.tsx

```diff
@@ -49,8 +48,8 @@ export const FireCentreFormControl = styled(FormControl)({
 const FireBehaviourAdvisoryPage: React.FunctionComponent = () => {
   const dispatch: AppDispatch = useDispatch()
   const { fireCenters } = useSelector(selectFireCenters)
-  const { hfiThresholdsFuelTypes } = useSelector(selectHFIFuelTypes)
-  const { fireZoneTPIStats } = useSelector(selectFireZoneTPIStats)
+  const { fireCentreTPIStats } = useSelector(selectFireCentreTPIStats)
```

Comment:
Done in 81aa4ac

Reply:
Though I guess the benefit of passing them to the child is easier testing, can just mock the object instead of mocking a redux store and state for each...